app/assets/stylesheets/lines.scss

```diff
@@ -1,3 +1,21 @@
 // Place all the styles related to the lines controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+input[type="radio"] {
```

Comment:
Can we nest the stuff in this stylesheet in a class, so it doesn't apply globally?

Reply:
yup, good call. It is all nested now